browser/components/MarkdownPreview.js

```diff
@@ -25,6 +27,7 @@ function decodeHTMLEntities (text) {
 const { remote } = require('electron')
 const { app } = remote
 const path = require('path')
+const dialog = remote.require('electron').dialog
```

Comment:
`electron` seems to be required twice.
If you import it on the first line, you don't need require twice.

```javascript
import { remote, dialog } from 'electron'
```